.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: bug'
+assignees: mastercoms
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**How you encountered the issue**
+Tell us the steps on how to recreate the bug.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+* Team Comtress version (don't say latest, say the actual version):
+* .NET Core version:
```

Comment:
We don't really care about .NET and DepotDownloader version. Issues with those things are not bugs, they are tech support questions

Reply:
done